custom_components/ppc_smgw/gateways/theben/thebenconnexa/thebenconnexa.py

```diff
@@ -33,7 +33,7 @@ async def get_data(self) -> Information:
         information = Information(
             firmware_version=await self._get_firmware_version(),
             readings=await self._get_readings(),
-            last_update=datetime.now(),
+            last_update=datetime.now(timezone.utc),
```

Comment:
Is this really in UTC or is this the in the local timezone?

In the PPC adapter I had to do the following:

https://github.com/jannickfahlbusch/ha-ppc-smgw/blob/0c7aad2c158e5bc667e854cb4d863a7152c61127/custom_components/ppc_smgw/ppcsmgw/ppc_smgw.py#L11
https://github.com/jannickfahlbusch/ha-ppc-smgw/blob/0c7aad2c158e5bc667e854cb4d863a7152c61127/custom_components/ppc_smgw/ppcsmgw/ppc_smgw.py#L155